### DISCUSSION TITLE
There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.

### DISCUSSION THREAD

**Discussion by yaider16** [community/community#114628] [New to GitHub]
Hi everybody, my subscription in students pack expired, and I wanted to make another request, but it's not letting me make another request, and it gives this message “There are outstanding issues with your GitHub account. Please contact GitHub Support for more information.” anyone knows how to resolve this? THX.

![Screenshot 2024-03-28 at 13-16-25 Discounted and free plans are available for educational use](https://github.com/community/community/assets/69011975/2823ead9-566f-47ed-ac8e-c6a04301b4cc)

**Comment by benogoulart** [comment]
Hi! This message usually indicates that there is a restriction or issue associated with your GitHub account. Unfortunately, only GitHub Support can determine the exact cause and remove the restriction.

Please contact [GitHub Support](https://support.github.com/contact) through the support portal and explain that you're unable to submit a new GitHub Student Developer Pack application because of this account issue.

They should be able to review your account and let you know what needs to be resolved.

Good luck!
